Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet.iOS/AppDelegate.cs

```diff
@@ -15,6 +16,13 @@ public override bool FinishedLaunching(UIApplication uiApplication, NSDictionary
             LoadApplication(new App());
             return base.FinishedLaunching(uiApplication, launchOptions);
         }
+
+        public override bool OpenUrl(UIApplication application, NSUrl url, string sourceApplication, NSObject annotation)
+        {
+            MSDistribute.OpenUrl(url);
```

Comment:
We don't document bindings, please use only the wrapper c# API we add in test apps.